views/userDetails.njk

```diff
@@ -311,6 +311,13 @@
             attributes: { "data-qa": "user-caseloads" }
           }) }}
 
+          {{ govukButton({
```

Comment:
Assume we need to check to only allow Central admins this link and same for removing a caseload.

Reply:
Yes - I have added to nomis-user-roles - now adding as part of HAAR-621